apps/barista-design-system/src/components/live-example/live-example.ts

```diff
@@ -142,14 +138,9 @@ export class BaLiveExample implements OnDestroy {
 
   /**
    * @internal
-   * The placeholder element that will be replaced with
-   * the example element once it is instantiated.
+   * The stream that holds the current component type.
    */
-  @ViewChild('demoPlaceholder', { read: ElementRef, static: true })
-  _placeholder: ElementRef;
-
-  /** @internal The component-ref of the instantiated class. */
-  _componentRef: ComponentRef<unknown>;
+  example$: Observable<any>;
```

Comment:
If marked as internal, please prefix.
```suggestion
  _example$: Observable<any>;
```